cachalot/__init__.py

```diff
@@ -1,4 +1,4 @@
-VERSION = (2, 3, 5)
+VERSION = (2, 3, 6)
```

Comment:
```suggestion
VERSION = (2, 4, 0)
```

Reply:
From the above comment about dropping version of Python and Django should change minor versions.